pkg/lib/deps/deps_provider.go

```diff
@@ -187,3 +208,12 @@ var BackgroundDependencySet = wire.NewSet(
 	clock.DependencySet,
 	globaldb.DependencySet,
 )
+
+var End2EndDependencySet = wire.NewSet(
+	appRootDeps,
```

Comment:
Instead of adding e2e stuff here, Make appRootDeps public and define End2EndDependencySet in the e2e package.

Reply:
I thought we are calling the api through http so we don't need additional dependencies? e.g. IP should be resolved based on the request.

Reply:
Yes the test itself part is simple http client 

This dependency set is for setting up project and user fixtures per test case

Reply:
So can we remove these changes?